src/main/java/isaac/bastion/listeners/BastionDamageListener.java

```diff
@@ -58,7 +65,52 @@ public void onBlockPlace(BlockPlaceEvent event) {
 			
 			event.setCancelled(true);
 			event.getPlayer().sendMessage(ChatColor.RED + "Bastion removed block");
+
+			damageRein(event);
+		}
+	}
+
+	private void damageRein(BlockPlaceEvent event)
```

Comment:
This entire method is complete bogus, why are you copying Citadel code into Bastion